### ISSUE TITLE
kgdoom (the lua scripting branch) causes a system crash on 5.1

### ISSUE THREAD

**Issue by convexVagabond** [kgsws/kgdoom#7]
After pressing the home button to exit the game, if you try to open any other applet or application, it crashes and forces a reboot 100% of the time

I used the latest (stable) version of libtransistor (v1.2.3) to compile it, and i'm using the latest version of hekate (as of ~8:30PM EST 6/6/2018), though i also tried it on the latest release of ctcaer's fork, which yielded the same results.

**Comment by kgsws** [comment]
Yes, there must be something that is not getting freed before exit. I do not know what is it yet.
(it crashes on 3.0.0 too)
